Functions.Templates/Templates/RabbitMQTrigger-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,71 @@
+{
+    "author": "Microsoft",
+    "classifications": [
+        "Azure Function",
+        "Trigger",
+        "RabbitMQ Queue"
+    ],
+    "name": "RabbitMQTrigger",
+    "identity": "Azure.Function.CSharp.RabbitMQTrigger.2.x",
+    "groupIdentity": "Azure.Function.RabbitMQTrigger",
+    "shortName": "rqueue",
+    "tags": {
+        "language": "C#",
+        "type": "item"
+    },
+    "sourceName": "RabbitMQTriggerCSharp",
+  "symbols": {
+    "namespace": {
+      "description": "namespace for the generated code",
+      "replaces": "Company.Function",
+      "type": "parameter"
+    },
+    "ConnectionStringSetting": {
+      "description": "The name of the app setting containing your RabbitMQ connection.",
+      "type": "parameter",
+      "defaultValue": "",
+      "replaces": "ConnectionValue"
+    },
+    "QueueName": {
+      "description": "This is the queue name.",
+      "type": "parameter",
+      "defaultValue": "myqueue",
+      "replaces": "NameOfQueue"
+    }
+  },
+    "primaryOutputs": [
+        {
+            "path": "RabbitMQTriggerCSharp.cs"
+        }
+    ],
+    "defaultName": "RabbitMQTriggerCSharp",
+    "postActions": [
+        {
+          "Description": "Adding Reference to Microsoft.Azure.WebJobs.Extensions.RabbitMQ Nuget package",
+          "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
+          "ContinueOnError": "true",
+          "ManualInstructions": [],
+          "args": {
+            "referenceType": "package",
+            "reference": "Microsoft.Azure.WebJobs.Extensions.RabbitMQ", "version": "1.0.0-beta",
+            "projectFileExtensions": ".csproj"
+          }
+        },
+        {
+          "description": "Restore NuGet packages required by this project.",
+          "manualInstructions": [],
+          "actionId": "210D431B-A78B-4D2F-B762-4ED3E3EA9025",
+          "continueOnError": true
+        },
+        {
+            "condition": "(HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
+            "description": "Opens the function class file in the editor",
+            "manualInstructions": [],
```

Comment:
does `actionId` have significance?

Reply:
I think they refer to the actual action that VS needs to do after the template is created. So yes, I think they do have significance. Have seen other examples as well, and they all have `actionId`